app/src/main/res/values-es/strings.xml

```diff
@@ -0,0 +1,118 @@
+    <!-- Eg: You have until 25/12/2022 (152) days left. -->
+    <string name="goal_days_left">Tienes hasta %s (%s) dias restante.</string>
+    <!-- Eg: You need to save around $2/day, $20/week, $200/month. -->
+    <string name="goal_approx_saving">Necesitas guardar alrededor de %s/</string>
+    <string name="goal_approx_saving_day">dia.</string>
+    <string name="goal_approx_saving_week">semana.</string>
+    <string name="goal_approx_saving_month">mes.</string>
+    <string name="goal_achieved_desc">Es tiempo de celebrar todo el trabajo que dirigió a esta alegre ocasión.</string>
+    <!-- Strings used for fragments for input screen -->
+    <string name="input_text_title">Entrar titulo</string>
+    <string name="title_empty_err">Woops! Titulo esta vacío.</string>
+    <string name="input_text_amount">Entrar cantidad</string>
+    <string name="amount_empty_err">Woops! Cantidad esta vacío.</string>
+    <string name="image_picker_input">Selecciona una imagen</string>
+    <string name="input_deadline">Entre fecha</string>
+    <string name="deadline_empty_err">Por favor entre fecha para esta meta!</string>
+    <string name="input_save_btn">Establece nueva meta de ahorro.</string>
+    <string name="input_edit_save_button">Modificar meta de ahorro.</string>
+    <string name="data_saved_success">Buen Trabajo! meta establecida.</string>
+    <string name="input_page_quote">No es necesario que veas el camino completo. Solo da el primer paso. – Martin Luther King, Jr.  </string>
+    <!-- info screen -->
+    <string name="info_end_date">Fecha Final</string>
+    <string name="info_total_amount">Cantidad de la meta</string>
+    <string name="info_current_amount">Ahorros Actuales</string>
+    <string name="info_remaining_amount">Cantidad Restante</string>
+    <string name="transactions">Transacciónes</string>
+      <string name="info_deposited">Depositado</string>
+    <string name="info_withdrawn">Retirado</string>
+    <!-- biometric lock title -->
+    <string name="bio_lock_title">Desbloquear GreenStash</string>
+    <string name="bio_lock_subtitle">Confirma tu bloqueo de pantalla, PIN or contraseña.</string>
+    <!-- Preference Titles -->
+    <string name="display_pref_header">Pantalla</string>
+    <string name="locale_pref_header">Locales</string>
+    <string name="security_pref_header">Seguridad</string>
+    <string name="about_pref_header">Sobre</string>
+    <!--  Display Preferences -->
+    <string name="display_pref_title">Tema Predeterminado</string>
+    <string name="material_you_perf_title">Habilitar Material You</string>
+    <string name="material_you_perf_summary">Cambia tema de acuerdo a tu fondo de pantalla (Requiere Android 12+).</string>
+    <string name="material_you_unavailable">Perdon, parece que Material You no esta soportado en su dispositivo o version de android.</string>
+    <!-- Locale Preferences -->
+    <string name="currency_pref_title">Moneda Predeterminada</string>`
```

Comment:
Looks like you've added a (`) here by mistake.

Reply:
on line 109? just fixed it.